core/client/kubernetes/resource.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/kiegroup/kogito-operator/core/client"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	client2 "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
why `client2` here and `runtime` in cmd package: https://github.com/kiegroup/kogito-operator/pull/995/files?diff=split&w=1#diff-7e8d72ebafe3281e9d914d78f8ede380eaf8df396a1bab4a6fa75c0fa543671bR25 ?

seems `runtime` would fit ...

Reply:
now at every place we are using `client` to represent `sigs.k8s.io/controller-runtime/pkg/client`